### PR TITLE
DEV-7846 Added in a notebook which simulates a fairly typical IBOR us…

### PR DESCRIPTION
**Pull request by MikeMcGarry**
…er journey

### CODE REVIEW

File: examples/use-cases/ibor/IBOR User Journey.ipynb [hunk 1]
```diff
@@ -0,0 +1,9434 @@
+{
```
Comment (rizwansaeed) [inline examples/use-cases/ibor/IBOR User Journey.ipynb:1]: <p>We need to move to an "upsert" style creation of transaction config (see <a href="https://github.com/finbourne/lusid-python-tools/blob/master/lusidtools/cocoon/transaction_type_upload.py" target="_blank">https://github.com/finbourne/lusid-python-tools/blob/master/lusidtools/cocoon/transaction_type_upload.py</a>)</p><p> </p><p>Reason being that we now have some pre-canned features that are demo-able in the demo client which rely on transaction config so there is a risk of breaking these if we replace the config with a static set</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (rizwansaeed) [inline examples/use-cases/ibor/IBOR User Journey.ipynb:1]: <p>I think you can use <code><strong>rename_properties=True</strong></code><strong><em> </em></strong> on <code>lusid_response_to_data_frame</code>  to give nicer column names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (rizwansaeed) [inline examples/use-cases/ibor/IBOR User Journey.ipynb:1]: <p>"upsert" transaction config</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (MikeMcGarry) [reply examples/use-cases/ibor/IBOR User Journey.ipynb:1]: It is set to True already in the function get_holdings, this is them looking good!

Comment (MikeMcGarry) [reply examples/use-cases/ibor/IBOR User Journey.ipynb:1]: Moved to upsert method.

Comment (MikeMcGarry) [reply examples/use-cases/ibor/IBOR User Journey.ipynb:1]: Moved the upsert method

**Pull request merged**